src/javascript/dom-work.js

```diff
@@ -0,0 +1,124 @@
+import layout from './layout';
+import getTemp from './api-data';
+import getCityImage from './city-image-api';
+
+const colorTemps = {
+  0: '#b3c6ff',
+  10: '#b3e4ff',
+  20: '#fff8b3',
+  30: '#ffecb3',
+  40: '#ffc6b3',
+};
+
+const elems = () => ({
+  body: document.body,
+  backgnd: document.querySelector('#backgnd'),
+  city: document.querySelector('#city'),
+  bttn: document.querySelector('#submit'),
+  title: document.querySelector('.title'),
+  pgBttm: document.querySelector('#bottom'),
+  error: document.querySelector('#error'),
+  errMssg: document.querySelector('.errMssg'),
+  date: document.querySelector('.date'),
+  flag: document.querySelector('#flag'),
+  cloud: document.querySelector('#cloud'),
+  sky: document.querySelector('#sky'),
+  rTemp: document.querySelector('.realTemp'),
+  degree: document.querySelector('#degrees'),
+  feels: document.querySelector('.feels'),
+  maxTemp: document.querySelector('.maxTemp'),
+  minTemp: document.querySelector('.minTemp'),
+  speed: document.querySelector('#speed'),
+  humid: document.querySelector('#humidity'),
+});
+
+const render = () => {
+  document.body.innerHTML = layout;
+  setTimeout(() => {}, 100000);
+  elems().backgnd.style.backgroundImage = 'url("https://cdn.pixabay.com/photo/2018/06/03/00/18/sea-3449509_1280.jpg")';
+
+  return new Promise((resolve) => {
+    if (elems().body.childElementCount > 1) {
+      resolve('Rendered successfully');
+    }
+  });
+};
+
+const makeNewDate = () => {
+  const Months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+  const date = new Date();
+  return `${Months[date.getMonth() + 1]} ${date.getDate()}, ${date.getFullYear()}`;
+};
+
+const updateInfo = (values, city) => {
```

Comment:
We can use objtect destructuring to get all the items in values object. For example, we can get
```js
const { TempDT, countryDt } = values;
```
We can then access `TempDT` and `countryDt` directly without using repeating `values `.
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/destructuring_requirement.png) from Pathwright).